local.py

```diff
@@ -43,7 +48,7 @@
 
 flags.DEFINE_enum(
     'mode', 'classify',
-    ['embed', 'classify', 'prefetch', 'thresholds', 'predict', 'csv', 'import'],
+    ['embed', 'classify', 'prefetch', 'thresholds', 'csv', 'import', 'train'],
```

Comment:
what happened to the predict mode?

Reply:
predict was only used to update the predicted_labels in the internal classification_sample database. 'classify' is used to get predictions for a sequence.

Since it turned out that syncing Uwazi to the classification_sample db was hard (e.g. we'd need to detect deleted entities and stop using them for training), I decided the whole TC<->Uwazi communication would be easier by removing the need for the classification_sample db. It can still be used for ./run local --mode=train etc., but Uwazi now sends the whole training data with the train HTTP request and uses the /classify endpoint for predictions, so there was no need for the 'update predictions' task anymore and I deleted it.